src/videojs-hls.js

```diff
@@ -239,7 +244,14 @@ videojs.Hls.prototype.src = function(src) {
   });
 };
 
-videojs.Hls.setMediaIndexForLive = function(selectedPlaylist) {
+/* Returns the media index for the live point in the current playlist, and updates
+   the current time to go along with it.
+ */
+videojs.Hls.getMediaIndexForLive = function(selectedPlaylist) {
+  if (!selectedPlaylist.segments) {
```

Comment:
Never in production, in theory, but we do it all the time in the unit tests. This is the breakage I was seeing after my first two commits.